server/controllers/todoController.js

```diff
@@ -0,0 +1,80 @@
+const pool = require("../database/connection");
+
+module.exports = {
+    /**
+    * get all todos from the database and return them
+     */
+    async getAll(req, res) {
+        try {
+            const allTodos = await pool.query("SELECT * FROM todo");
+            res.status(200).json(allTodos.rows);
+        } catch (error) {
+            console.error(error.message);
+        }
+    },
+
+    /**
+     * get a todo from the database and return it
+     */
+    async getOne(req, res) {
+        try {
+            const { id } = req.params;
+            const getTodo = await pool.query(
+                "SELECT * FROM todo WHERE todo_id = $1",
+                [id]
+            );
+            res.json(getTodo.rows[0]);
+        } catch (error) {
+            console.error(error.message);
+        }
+    },
+
+    /**
+     * add a todo to the database and return it
+    */
+    async add(req, res) {
+        try {
+            const { description } = req.body;
+            const newTodo = await pool.query(
+                "INSERT INTO todo (description) VALUES($1) RETURNING *",
+                [description]
+            );
+            res.json(newTodo.rows[0]);
+        } catch (error) {
+            console.error(error.message);
+        }
+    },
+
+    /**
+     * update a todo in the database 
+     */
+    async update(req, res) {
+        try {
+            const { id } = req.params;
+            const { description } = req.body;
+            const updateTodo = await pool.query(
```

Comment:
remove unused variebles

Reply:
Removed unused variables in commit 31ac9b9a760b3526e5c2a0f370e926764cc0409d to branch mui-components